packit/api.py

```diff
@@ -2343,11 +2347,27 @@ def run_osh_build(
         if csmock_args:
             cmd.append("--csmock-args=" + shlex.quote(csmock_args))
 
-        cmd.append("--config=" + str(chroot))
+        osh_options = self.package_config.osh_options
+
+        if osh_options is not None:
```

Comment:
There's like 30 lines of code in between… I would probably recommend going with the walrus operator, I'll provide an example below.